src/table/thead.tsx

```diff
@@ -3,12 +3,12 @@
 import React from 'react';
 import clsx from 'clsx';
 
-import { findUpUntil } from '@cloudscape-design/component-toolkit/dom';
```

Comment:
The fix for component-toolkit version of the utility: https://github.com/cloudscape-design/component-toolkit/pull/99